lib/Advisor.js

```diff
@@ -174,24 +177,27 @@ class Advisor {
     tabs.off('close', this.hidePanel);
   }
 
-  addRec(domain) {
-    const rec = {
-      domain,
-    };
-    simpleStorage.storage.recData.push(rec);
-  }
-
-  loadOwnedDomains() {
+  loadAPIData() {
     const req = request({
       url: 'https://haveibeenpwned.com/api/v2/breaches',
       onComplete: response => {
         for (let site of response.json) { // eslint-disable-line prefer-const
-          this.addRec(site.Domain);
+          if (!simpleStorage.storage.recData[site.Name]) {
+            simpleStorage.storage.recData[site.Name] = site;
```

Comment:
What if the `recData` changed on the site? Wouldn't it be better to always overwrite the local storage?